pages/faq/troubleshooting/faq.md

```diff
@@ -138,6 +139,10 @@ Discontinued devices will no longer be actively supported by {{ $names.company.l
 
 There are a few options for devices that do not have an official device type on {{ $names.company.lower }}. If your device has an x86 architecture, you can try the [Intel NUC][nuc] image, which is built to support generic x86 devices. For other devices, you can [build your own][build-your-own] version of {{ $names.os.lower }} using our [open source repos][balenaOS]. To discuss custom board support, please contact sales@{{ $names.email_domain }}.
 
+##### What to keep in mind when choosing power supply units?
+
+Power supply units (PSUs) are a critical component to any widespread production deployment, as mentioned in our [Going to Production][go-to-production] guide. When buying a PSU for your hardware, be sure to inspect and understand the board's power requirements carefully. Current requirements (measured in Amps) are just as important as the voltage requirements (measured in Volts). A PSU that provides a higher voltage than required may damage the board. Still, a supplied voltage that is too low may cause the board to malfunction or fail to boot if the PSU cannot provide the maximum current that the board demands. The board might eventually fail to perform under CPU-intensive loads. As the load increases, more current is drawn than the PSU can supply, which could lead to brown-outs (i.e., the board unexpectedly resetting, causing filesystem corruption). When buying PSUs or using the official power adapters, proper research is recommended to prevent scenarios like these in a production deployment.
```

Comment:
You previously had separate sentences about voltage and current; but you seem to have merged them into one sentence and I'm not sure if it makes sense now? :shrug: 
"Still, a supplied voltage that is too low may cause the board to malfunction or fail to boot if the PSU cannot provide the maximum current that the board demands."

Entirely up to you of course though! :+1:

Reply:
@lurch I agree, grammar checker scrambled the sentences in an effort to reduce the length of the sentence. Reverted, should be good to go now.